todolist/settings.py

```diff
@@ -62,10 +62,15 @@
 # https://docs.djangoproject.com/en/1.7/ref/settings/#databases
 
 DATABASES = {
-    "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
+    'default': {
+        'ENGINE': 'mysql.connector.django',
```

Comment:
The database engine 'mysql.connector.django' might not be compatible with Django 1.7. Consider using 'django.db.backends.mysql' if you encounter compatibility issues.